### PR TITLE
Adds the RD hardsuit from TG, removes war crimes

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

This PR adds the RD hardsuit, a hardsuit with a diagnostic hud built in, the ability to check reagents like science goggles, and able to detect nearby explosions like a tachyon array.

Also removes the red cross from some clothes that still have them, replaced with a green cross

##Todo
Sprites
- [x] fix missing pixels on drask
- [x] vox
- [x] unathi
- [x] vulp
- [x] tarjan
- [x] skrell
- [x] grey

## Why It's Good For The Game

War crimes are bad.

Rd could use a hardsuit, like the ones the other heads get, and this PR adds it. Not good for combat, with armor basicaly the same as any other hardsuit, with just as much slowdown. Sure, it is something armored the rd could wear, and theoretically speed potion, but they already have their RD armor. All command has EVA access as well, and with the med maint and science remap, the RD has a SSU in their office, with a normal EVA suit, so adding this will not really have a dangerous impact.

## Images of changes

Lights are on on hardsuit in photos
image

Grey, light off, light on, needs to be shifted up 1 pixel with the lighted helmet in this picture.
image

Skrell, slight innacuracy with middle image which was fixed after
image

Tajaran
image

Vulp, 1 leaking pixel at the top that was fixed after
image

Unathi 🦎 
image

Vox 🐦 
image

See icon diff for the war crime changes, and updated sprites, some of the older ones have been adjusted slightly

## Changelog
:cl:
add: Adds the RD's hardsuit, a bomb and acid resistant hardsuit, perfect for dealing with bombs in style.
tweak: Also removes war crimes (red cross) from more armors / suits for all species.
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Finally all the heads will have hardsuits.

File: code/game/objects/explosion.dm [hunk 1]
```diff
@@ -192,9 +192,11 @@
 		//Machines which report explosions.
 		for(var/i,i<=GLOB.doppler_arrays.len,i++)
 			var/obj/machinery/doppler_array/Array = GLOB.doppler_arrays[i]
+			var/obj/item/clothing/head/helmet/space/hardsuit/rd/Helm_Array = GLOB.doppler_arrays[i]
```
Comment (SteelSlayer) [inline code/game/objects/explosion.dm:195]: This is a direct cast, there is no type checking here, so both `Helm_Array` and `Array` will be the same object on each iteration of the loop. Meaning that whatever `GLOB.doppler_arrays[i]` is, will have two calls to `sense_explosion()`.

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -491,6 +491,67 @@
...
 
+	//Research Director hardsuit
+/obj/item/clothing/head/helmet/space/hardsuit/rd
+	name = "prototype hardsuit helmet"
+	desc = "A prototype helmet designed for research in a hazardous, low pressure environment. Scientific data flashes across the visor."
+	icon_state = "hardsuit0-rd"
+	item_state = "rd"
+	item_color = "rd"
+	flash_protect = 0
+	scan_reagents = 1
```
Comment (SteelSlayer) [inline code/modules/clothing/spacesuits/hardsuit.dm:502]:
```suggestion
	scan_reagents = TRUE
```

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 3]
```diff
@@ -491,6 +491,67 @@
...
+
+
+/obj/item/clothing/head/helmet/space/hardsuit/rd/dropped(mob/living/carbon/human/user)
+	..()
+	if((user.head == src) && hud_active)
+		GLOB.doppler_arrays -= src
+		var/datum/atom_hud/H = GLOB.huds[DATA_HUD_DIAGNOSTIC]
+		H.remove_hud_from(user)
+
+/obj/item/clothing/head/helmet/space/hardsuit/rd/proc/sense_explosion(var/x0,var/y0,var/z0, var/devastation_range, var/heavy_impact_range,
```
Comment (SteelSlayer) [inline code/modules/clothing/spacesuits/hardsuit.dm:527]: Implicit `var/` on all these arguments.

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 4]
```diff
@@ -491,6 +491,67 @@
...
+		var/light_impact_range, var/took, var/orig_dev_range, var/orig_heavy_range, var/orig_light_range)
+	var/turf/T = get_turf(src)
+	var/dx = abs(x0-T.x)
+	var/dy = abs(y0-T.y)
```
Comment (SteelSlayer) [inline code/modules/clothing/spacesuits/hardsuit.dm:531]:
```suggestion
	var/dx = abs(x0 - T.x)
	var/dy = abs(y0 - T.y)
```

**Comment by Aracino** [comment]
Now to jack this in case I forget to set the receiver on my toxin bombs.

File: code/game/objects/explosion.dm (hunk 1 above)
Comment (Qwertytoforty) [reply code/game/objects/explosion.dm]: Fixed with your help!

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 3 above)
Comment (Qwertytoforty) [reply code/modules/clothing/spacesuits/hardsuit.dm]: Removed

**Comment by TDSSS** [comment]
Eh. Why does science need a buff, again? Especially the RD, who has access to all of its tools, as well as every command area? I don't see why heads without any business in space need their own hardsuit.

**Comment by procdrone** [comment]
> 
> 
> Eh. Why does science need a buff, again? Especially the RD, who has access to all of its tools, as well as every command area? I don't see why heads without any business in space need their own hardsuit.

You say this but when the toxins or scichem blows up half of science again you will wish you had it :D

**Comment by DeveloperLoser** [comment]
> Eh. Why does science need a buff, again? Especially the RD, who has access to all of its tools, as well as every command area? I don't see why heads without any business in space need their own hardsuit.

Or when their max cap bounces off a coi.

**Review by Kyep — APPROVED** [review]

**Comment by MrMagolor** [comment]
0/10 no drask sprites.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**